modules/tensor_mechanics/doc/content/source/actions/MechanicalBodyForceAction.md

```diff
@@ -0,0 +1,8 @@
+# Mechanical Body Force Action
+
+## Description
+
+The `MechanicalBodyForce` Action is used to create a set of body force kernels
+for the displacement variables of the mechanical problem. See the description,
```

Comment:
Should probably add something here about the fact that the "set" consists of one instance per coordinate direction.